library/package.json

```diff
@@ -89,11 +89,11 @@
     "globals": "^16.0.0",
     "graphql": "^16.8.2",
     "hono": "^4.4.2",
-    "koa-router": "^12.0.1",
+    "koa-router": "^13.0.1",
     "koa-v2": "npm:koa@^2.16.1",
     "koa-v3": "npm:koa@^3.0.0",
     "mariadb": "^3.3.2",
-    "mongodb": "~6.9",
```

Comment:
this was a version that lazy loaded the collection I think? we can safely upgrade?

Reply:
You fixed that in #471?